### PR TITLE
PS-3812 Search widget for concrete products 

### PR DESCRIPTION
**Pull request by Compixx**
- Developer(s): @Compixx

- Ticket: https://spryker.atlassian.net/browse/PS-3812

- Academy PR: -

- Release Group: URL_HERE

#### Please confirm

- [x] No new OS components - or they have been approved by legal department

#### Documentation

- [x] Functional documentation provided or in progress?
- [x] Integration guide for projects provided or not needed?
- [x] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   EventBehavior         | minor                 |                       |

#### Release Notes

{skip}

-----------------------------------------

##### Change log

Improvements

- Introduced `EventBehaviorFacade::getGroupedEventTransferForeignKeysByForeignKey()` to group foreign keys from `eventTransfer` by foreign key value.

### CODE REVIEW

File: tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] fc646900bab47048b53a79d3a36c8cf7
+<?php  //[STAMP] 6543653be52273f4c175eced1304a144
```
Comment (dereuromark) [inline tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php]: Shouldnt this file be not commited in git??

Comment (gerner-spryker) [reply tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php]: I have no idea. @ehsanmx ?

Comment (ehsanmx) [reply tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php]: @dereuromark yes you are right, these files are generated files and should not be committed into git

Comment (dereuromark) [reply tests/SprykerTest/Zed/EventBehavior/_support/_generated/EventBehaviorBusinessTesterActions.php]: You can make follow up fix PR (merge only)

**Pull request merged**